core/src/main/scala/akka/persistence/cassandra/query/EventsByTagStage.scala

```diff
@@ -246,7 +251,9 @@ import scala.compat.java8.FutureConverters._
       }
 
       override def preStart(): Unit = {
-        stageState = StageState(QueryIdle, fromOffset, calculateToOffset(), initialTagPidSequenceNrs, None, bucketSize)
+        stageState = StageState(QueryIdle, fromOffset, calculateToOffset(), initialTagPidSequenceNrs.mapValues {
```

Comment:
I think we should avoid `mapValues` since it's a view

Reply:
Yup, will switch to transform